RadixWallet/Features/ImportMnemonic/ImportMnemonic.swift

```diff
@@ -659,48 +647,15 @@ extension ImportMnemonic {
 		}
 	}
 
-	private func focusNext(_ state: inout State) -> Effect<Action> {
-		if let current = state.idOfWordWithTextFieldFocus {
+	private func focusNext(_ state: inout State, after current: Int?) -> Effect<Action> {
+		if let current {
 			state.words[id: current]?.resignFocus()
-		}
-		guard let nextID = state.words.first(where: { !$0.isComplete })?.id else {
+			return delayedEffect(delay: .milliseconds(75), for: .internal(.focusOn(current + 1)))
```

Comment:
the `delayedEffect` contains a bit of duplication IMO, maybe can be changed to something like?:

```swift
if let target = current.map { $0 + 1 } ?? state.words.first(where) {
    return delayedEffect(delay: .milliseconds(75), for: .internal(.focusOn(target)))
}
```

Reply:
So what you want is to merge the two calls two `delayedEffect` into one that handle both cases? In such case you wouldn't be performing line 652
```Swift
state.words[id: current]?.resignFocus()
```

Reply:
should be handled ofc :) 

```swift
var target: ImportMnemonicWord.State.ID?
if let current { ... resignFocus(); target = .. }
if let target { ... }
```

dont feel to strongly about it, otherwise you can create a variable for the `75` milliseconds? or use the predefined? `delayedEffectShort` or `delayedShortEffect` or whats it called?

Reply:
It ends up being a method with 12 lines of code instead of 8.

The `delayedMediumEffect` & `delayedShortEffect` take longer time (0.6s & 0.3s), while with 75ms this works already fine. I would leave as is tbh